index.html

```diff
@@ -286,8 +286,9 @@ <h1>Contact me</h1>
                     <li>
                         <textarea name="message" id="msg" maxlength="500" placeholder="Enter Your Message Here" required></textarea>
                     </li>
-                    <li>
-                        <button type="submit">Get In Touch</button>
+                    <li class="button-p">
+                        <button class="submit-btn" type="submit">Get In Touch</button>
+                        <p class="error-message"></p>

```

Comment:
- [ ] It would be better if you display the `error` message above the `submit` button because, on each message display, this affects the form design and change the button position. It's a little change, I know you can do that 😊

**Check out how it looks:**

![image](https://user-images.githubusercontent.com/80612925/175577212-dd7be454-7b36-46e0-be1c-b8f3868f0128.png)